minecraft_servers/syntaxtnt/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "server_name": "syntaxtnt",
+  "nice_name": "SyntaxTNT",
+  "direct_ip": "syntaxtnt.de",
+  "server_wildcards": [
+    "%.syntaxtnt.de"
+  ],
+  "social": {
+    "web": "https://syntaxtnt.de",
```

Comment:
Website not finished yet. Please fix or remove.
![Bildschirmfoto 2023-07-24 um 08 20 21](https://github.com/LabyMod/server-media/assets/45363287/1670af9d-d83e-4ecb-9c2c-4d39e95b3ec4)